src/Ninja/G1CP/Content/patchInit.d

```diff
@@ -63,6 +63,7 @@ func void Ninja_G1CP_Menu(var int menuPtr) {
         G1CP_122_CavalornDailyRoutine();                // #122
         G1CP_125_ButcherText();                         // #125
         G1CP_126_SharkyTrade();                         // #126
+        G1CP_133_DE_LogEntryScatty();                   // #133
```

Comment:
This is a gamesave fix. It needs to be initialized from `G1CP_GamesaveFixes_Apply` and reverted in `G1CP_GamesaveFixes_Revert` not here.